File: ModeratorServer/src/main/java/dk/groupfive/ModeratorServer/model/objects/ReviewLite.java
package dk.groupfive.ModeratorServer.model.objects;

import java.util.List;

public class ReviewLite implements IReview {
    private double rating;

    public ReviewLite(double rating) {
        this.rating = rating;
    }

    @Override
    public double getRating() {
        return rating;
    }

    @Override
    public List<ReviewItem> getReviews() {
        return null;
    }
}

File: Client/Client/Data/ReviewLite.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Client.Data
{
    public class ReviewLite : IReview
    {
        double rating;
        public double GetRating()
        {
            return rating;
        }

        public List<ReviewItem> GetReviews()
        {
            return null;
        }
    }
}
